### ISSUE TITLE
I want to know what models files are

### ISSUE THREAD

**Issue by xiaoshuguo750** [tnikolla/robot-grasp-detection#8]
Hi!

I want to know what models files are,and how can I use them?

Thank you very much!

**Comment by tnikolla** [comment]
Hi,

these are files with the variables trained (optimized) from imagenet classification algorithm and grasp-detection algorithm. You can use them by giving them as arguments, for example:

    ./imagenet_classifier.py --batch_size=128 --model_path=./models/imagenet/m1/m1.ckpt --train_or_validation=train

Give your path to --model_path=your/path/to/the/ckpt_file.

For a better explanation of what they are and how to use them see [tensorflow guide](https://www.tensorflow.org/programmers_guide/saved_model).

**Comment by HEUzhouhanwen** [comment]
Thank you for your answer!
When I run the following command（using Cornell Grasping Dataset）, there are some problems（Step 23 | loss = nan）, what is the reason? What should I do?

 ./grasp_det.py --batch_size=64 --model_path=./models/imagenet/m2/m2.ckpt --train_or_validation=train
some mistake happened!what's wrong?

Step 21 | loss = 177726.0
 | x = [  92.57499695  107.09999847   77.875     ]
 | x_hat = [ 71.45228577  70.01683807  78.42079163]
 | tan = [ 0.04202652 -0.11677019 -8.66753483]
 | tan_hat = [-15.22892761   7.36084843 -13.21743774]
 | h = [ 18.9166832   16.20939255  33.59136581]
 | h_hat = [  8.36284542  10.77396488  17.70290947]
 | w = [ 15.04407215   9.35572624  12.16120529]
 | w_hat = [ 21.95122147  11.86730862   8.71802139]
 | (0.983 sec/batch


Step 22 | loss = 145800.0
 | x = [ 113.22499847  101.32499695  111.29999542]
 | x_hat = [ 88.18299866  69.44770813  81.33192444]
 | tan = [-1.64852488        -inf -0.33319956]
 | tan_hat = [  1.98662510e+01  -2.18759489e+00  -1.49553716e-02]
 | h = [ 26.99363327  19.73999977  20.29046822]
 | h_hat = [ 17.33011055   9.94925213  21.10087013]
 | w = [ 10.8564539    6.64999962  13.38611507]
 | w_hat = [ 19.93681526   0.93864435  17.20625496]
 | (0.963 sec/batch


Step 23 | loss = nan
 | x = [  94.84999847   93.79999542  122.5       ]
 | x_hat = [ 89.2689743   98.40085602  95.08123016]
 | tan = [ 1.84252501 -9.06495476 -0.        ]
 | tan_hat = [-6.25003433  3.59752798 -0.40878177]
 | h = [ 31.55083656  19.15188408   6.29999876]
 | h_hat = [ 24.60993385  24.8934269    7.01266623]
 | w = [ 11.33346653   9.29216862   3.75999999]
 | w_hat = [ 26.53094292  20.04769325  22.91687965]
 | (0.975 sec/batch


Step 24 | loss = nan
 | x = [ 107.97499847  108.5          99.57499695]
 | x_hat = [ nan  nan  nan]
 | tan = [-0.38987583  0.43599257 -1.42756033]
 | tan_hat = [ nan  nan  nan]
 | h = [ 11.26979828  29.40008545  31.7219696 ]
 | h_hat = [ nan  nan  nan]
 | w = [  8.17940044  31.39706039  10.76076698]
 | w_hat = [ nan  nan  nan]
 | (0.964 sec/batch

Thank you very much!

**Comment by tnikolla** [comment]
Hi,

check first issue #4 .

**Comment by xiaoshuguo750** [comment]
Thank you！

At 2018-01-01 18:00:59, "Tomi Nikolla" <notifications@github.com> wrote:


Hi,

check first issue #4 .

**Comment by zhang-lu153** [comment]
Hi, Can you give your ./models/imagenet/m1/m1.ckpt ?I want to know what this models file is
My email is zl@hnu.edu.cn .
Thank you!

**Comment by jinhuan-hit** [comment]
@lubao1995 You can see ./models/imagenet/m2/m2.ckpt in your path，not m1.
